sample-app/src/components/InputDropdown.tsx

```diff
@@ -14,7 +14,7 @@ interface Props {
   screenReaderInstructions: string,
   screenReaderInstructionsId: string,
   screenReaderText: string,
-  onlyAllowDropdownOptionSubmissions: boolean,
+  onlyAllowDropdownOptionSubmissions?: boolean,
```

Comment:
Out of scope of your PR, but what does this prop do? The name is somewhat confusing.

Reply:
This is from the FilterSearch pr, it's so that you can only submit options from the autocomplete dropdown, and not any arbitrary input ([I was also confused by it](https://github.com/yext/answers-headless-react/pull/64#discussion_r759509677))